### DISCUSSION TITLE
keyball39 ディスプレイが砂嵐の様になる、キーボード入力ができない

### DISCUSSION THREAD

**Discussion by poporirinu** [Yowkees/keyball#582] [Q&A]
お世話になっております。
以下現象について解決策をご教授頂けますでしょうか。

・他のケーブルを使用しても変わらずです

①左手側のPicoMicroにUSB接続・左右のTRSケーブル接続なし
　　→デバイスの脱着を連続で行ったような音が鳴る。
　　　左手側のキーボードのキー入力は受け付けない。
　　　OLEDの表示は正常。
![image](https://github.com/Yowkees/keyball/assets/160295919/cf9c9a8d-7693-4ded-87ef-06dd1ae6a8e0)![image](https://github.com/Yowkees/keyball/assets/160295919/47ee4036-5d09-45ff-8137-d06564c33708)

②右側PicoMicroにUSB接続・左右のTRSケーブル接続なし
　　→右手側キーボード・トラックボールは正常に使える
　　　OLEDの表示は正常。

③左手側のPicoMicroにUSB接続・左右のTRSケーブル接続あり
　　→デバイスの脱着が連続で発生するような認識がされる。
　　　全てのキーボードのキー入力は受け付けない。
　　　OLEDの表示は左右ともに正常。

④右手側のPicoMicroにUSB接続・左右のTRSケーブル接続あり
　　→左手側のOLEDの表示が砂嵐のようにおかしくなる。
　　　右手側のOLEDの表示は正常。
　　　左手側のキーボードは入力を受け付けない。
　　　右手側のキーボード、トラックボールは正常に使える。![image](https://github.com/Yowkees/keyball/assets/160295919/16d2b3bc-1381-490d-8501-30ab2568d645)![image](https://github.com/Yowkees/keyball/assets/160295919/ed4fbde8-2477-45d4-8796-4eddbdfce853)

![image](https://github.com/Yowkees/keyball/assets/160295919/6648c0c7-cb7d-4870-ac64-111dbbfc6dad)![image](https://github.com/Yowkees/keyball/assets/160295919/090b83d7-2ccd-4dce-b8e2-ac70604fd6e9)![image](https://github.com/Yowkees/keyball/assets/160295919/a0cc74ee-db4c-45cf-b86a-b510642279b1)![image](https://github.com/Yowkees/keyball/assets/160295919/4616ee37-c743-41d3-9b46-5df67a3adfce)

**Comment by Ryo624** [comment]
ProMicroに書き込むファームウェアがremapのKeyball39_viaを書き込んでも同じような症状は出ますでしょうか。

**Reply by poporirinu** [reply]
無事なおりました。
ご回答ありがとうございました！
